### PR TITLE
Feature/upgrade light functionality

### PR DESCRIPTION
**Pull request by eerison**
a new parameter was introduced to handle lights

### CODE REVIEW

**Comment by eerison** [comment]
@dbuezas do you know any easy way to test this change with my HA 🤔 ?

**Comment by dbuezas** [comment]
Hi :wave: yes it is quite easy to test, just copy the files you modified to your home assistant computer, inside config/custom_components/icsee-ptz. Then restart home assistant.

I suggest you try the samba mount add on so you can access the home assistant folders directly from your pc as a network drive.

https://github.com/home-assistant/addons/blob/master/samba/DOCS.md

File: README.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ streams:
 | Entity                                    | Description                                                              | Enabled by default |
 | ----------------------------------------- | ------------------------------------------------------------------------ | ------------------ |
 | `switch.*_blinddetect_enabled`            |                                                                          | No                 |
-| `select.*_day_night_color`                | You can control camera's lights.                                         | No                 |
+| `select.*_whith_light`                    | You can control camera's lights.                                         | No                 |
```
Comment (eerison) [inline README.md:168]: Hey @dbuezas 

where is it defined?
my current code in select.py is still using the old name `select.camera_varanda_day_night_color`.

Comment (eerison) [reply README.md:168]: Ok after I restart, it is the new code 😄

Comment (dbuezas) [reply README.md:168]: Try removing and reading the cameras

Comment (dbuezas) [reply README.md:168]: Oh, yes, you need to restart after any code change, no exceptions 👍

Comment (eerison) [reply README.md:168]: Now it is showing on entities.
Screenshot 2023-12-19 at 11 39 52

But not here:
Screenshot 2023-12-19 at 11 40 10

Any idea why? am I missing something?

Comment (eerison) [reply README.md:168]: I found the reason 😄

**Comment by eerison** [comment]
@dbuezas i guess the code is good to review :)

**Comment by dbuezas** [comment]
Cool, thanks!
Can you readd the deleted one? The cameras are a bit like the wild west, mine worked with the old config

**Comment by eerison** [comment]
In theory your camera should work, because it is the new parameter that icsee app is using

**Comment by eerison** [comment]
I have a camera that work with old config, and this camera worked with this new parameter as well,  for this reason I guess it should also work for you

**Comment by dbuezas** [comment]
As I said, it is the wild west.
I don't update cameras anymore because some started using their custom video protocol and go2rtc doesn't work well with it.

**Comment by dbuezas** [comment]
Lets keep both for a while :)

**Comment by eerison** [comment]
> Lets keep both for a while :)

Should I create a new select for this?

File: custom_components/icsee_ptz/select.py [hunk 2]
```diff
@@ -26,45 +26,30 @@ async def async_setup_entry(
 
     async_add_entities(
         [
-            DayNightColorSelect(hass, entry, channel)
+            WhiteLightSelect(hass, entry, channel)
```
Comment (eerison) [inline custom_components/icsee_ptz/select.py:29]: To keep both I keep both methods here??

**Comment by eerison** [comment]
Could you check again @dbuezas please :)

**Review by dbuezas — APPROVED** [review]
Looks good to me :👍

**Comment by dbuezas** [comment]
I don't have a computer for some weeks but it looks good!
Thanks for the contribution!

**Comment by eerison** [comment]
> I don't have a computer for some weeks but it looks good! Thanks for the contribution!

when you have some time could you generate a `release` or `pre-release` 👀

**Comment by dbuezas** [comment]
Done (pre-release)

**Pull request merged**